frontend/packages/ceph-storage-plugin/src/components/ocs-install/ocs-request-data.ts

```diff
@@ -140,6 +141,15 @@ export const getOCSRequestData = (
     },
   };
 
+  if (kmsEnable) {
+    requestData.spec.encryption = {
+      ...requestData.spec.encryption,
+      kms: {
+        enable: true,
+      },
+    };
+  }
```

Comment:
Just for consistency, can you use Object.assign as well similar to arbiter and network?